driver.go

```diff
@@ -4,6 +4,7 @@ import (
 	"github.com/pion/mediadevices/pkg/driver"
 	_ "github.com/pion/mediadevices/pkg/driver/camera"
 	_ "github.com/pion/mediadevices/pkg/driver/microphone"
+	_ "github.com/pion/mediadevices/pkg/driver/screen"
```

Comment:
BTW, it maybe better to load drivers by user code as same as codecs to minimize the binary size and number of dependencies according to the specific usage.

Reply:
I actually thought about that. But, I wasn't sure if it was worth it. Because if we do it like this, we'll add some complexities to the users, they need to know what to import for the getUserMedia to work properly which can be somewhat unintuitive especially for Go beginners.

Reply:
You actually brought some good points here, binary size and number of dependencies. Love it! I'll create a PR for this.